torch/_inductor/virtualized.py

```diff
@@ -823,6 +823,42 @@ def __mod__(self, other):
     def __pow__(self, other):
         return ops.pow(self, other)
 
+    def __lt__(self, other):
+        return ops.lt(self, other)
+
+    def __le__(self, other):
+        return ops.le(self, other)
+
+    def __eq__(self, other):
```

Comment:
It looks like defining equality has uncovered the fact I was creating a set of `OpsValue` which without `__eq__` being defined  was defaulting to identity equality which would never pass because we create new `OpsValue` wrappers even if cse happens. 

I think the cleanest fix is in `torch/_inductor/codegen/triton.py` you can just do
```python
from torch._inductor.virtualized import _ops as ops
```
which will call the underlying ops handler directly without creating `OpsValue` wrappers.